### PR TITLE
Release

### PR DESCRIPTION
**Pull request by BasvRossem**

### CODE REVIEW

File: module/camera_properties.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    def get_focal_length(self):
+        """
+        Gets the focal length
+        :return: Returns the focal length of the camera in millimeters
+        """
+        return self.focal_length
+
+    def get_sensor_height(self):
+        """
+        Gets the sensor height
```
Comment (florianhumblot) [inline module/camera_properties.py:54]: I don't think it gets the sensor height, please change

**Review by florianhumblot — CHANGES_REQUESTED** [review]
Looking pretty good, only have 1 comment to change and it'll be good to merge :)

File: main.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+"this module is a template"
```
Comment (gerritvanos) [inline main.py]: this should be removed i think since this is now a module and not a template anymore

**Review by gerritvanos — CHANGES_REQUESTED** [review]
looks fine only one comment and the comment of florian

File: module/camera_properties.py (hunk 1 above)
Comment (BasvRossem) [reply module/camera_properties.py:54]: Fixed in https://github.com/R2D2-2019/vision/commit/d473d24737d5ffb7655a90050e2977153ae85cad on line 61

File: main.py (hunk 2 above)
Comment (BasvRossem) [reply main.py]: Fixed in https://github.com/R2D2-2019/vision/commit/d473d24737d5ffb7655a90050e2977153ae85cad

**Review by gerritvanos — APPROVED** [review]

**Review by florianhumblot — APPROVED** [review]

**Pull request merged**